src/scripts/scenes/PlayerMapScene.ts

```diff
@@ -33,6 +35,9 @@ export class PlayerMapScene extends ViewportScene {
    private _path!: SmoothGraphics;
 
    override onLoad(): void {
+      const goBackToCity = () => Singleton().sceneManager.loadScene(WorldScene);
+      useShortcut("GoBackToCity", goBackToCity, []);
```

Comment:
React Hook should only be used in React Component, otherwise it will not get cleaned up properly, causing ghost shortcut